packages/strapi-design-system/src/EmptyStateLayout/EmptyStateLayout.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+
+import { Box, BoxProps } from '../Box';
+import { Flex, FlexProps } from '../Flex';
+import { Typography } from '../Typography';
+
+export interface EmptyStateLayoutProps extends FlexProps {
```

Comment:
This isn't really extending `FlexProps` it's more "picking" from them, because we never pass the rest of the spread to the node imo